docs/VnVPlan/VnVPlan.tex

```diff
@@ -69,141 +69,157 @@ \section{Symbols, Abbreviations, and Acronyms}
 
 \subsection{SRS Verification Plan}
-
-\wss{List any approaches you intend to use for SRS verification.  This may include
-  ad hoc feedback from reviewers, like your classmates, or you may plan for 
-  something more rigorous/systematic.}
-
-\wss{Maybe create an SRS checklist?}
+The design documentation will be verified with an informal walk-through by stakeholders (individuals who are members of low-income households in Hamilton and students from McMaster University) and peers of the SFWRENG 4G06 course who will provide ad-hoc review as well as an inspection by the development team. The inspection will address the following issues and be tracked by the checklist:
+\begin{itemize}
+    \item Correctness: the SRS accurately represents the software expectations
+    \item Ambiguity: the requirements and contents convey an unambiguous meaning
+    \item Completeness: the SRS covers all necessary functional and non-functional requirements
+    \item Verifiability: the requirements are testable and quantifiable
+    \item Traceability: the requirements have clear source references
+    \item Clarity: the document is written in a clear, concise, and unambiguous manner
+\end{itemize}
 
 \subsection{Design Verification Plan}
-
-\wss{Plans for design verification}
-
-\wss{The review will include reviews by your classmates}
-
-\wss{Create a checklists?}
+The SRS will be verified with an informal walk-through by stakeholders (individuals who are members of low-income households in Hamilton and students from McMaster University) and peers of the SFWRENG 4G06 course who will provide ad-hoc review as well as an inspection by the development team. The review will address the following issues and be guided by the checklist:
+\begin{itemize}
+    \item Design adequacy and conformance to software requirements: assess whether the design adequately meets the specified requirements (SRS)
+    \item Internal consistency: different components of the design are coherent
+    \item External consistency: the design aligns with external interfaces, standards, and environments
+    \item Feasibility: whether the design can be practically realized within technical and resource constraints
+    \item Maintenance: whether the design can be easily updated, modified, or repaired
+\end{itemize}
 
 \subsection{Verification and Validation Plan Verification Plan}
-
-\wss{The verification and validation plan is an artifact that should also be
-verified.  Techniques for this include review and mutation testing.}
-
-\wss{The review will include reviews by your classmates}
-
-\wss{Create a checklists?}
+The VnV plan will be verified with an informal walk-through by stakeholders (individuals who are members of low-income households in Hamilton and students from McMaster University) and peers of the SFWRENG 4G06 course who will provide ad-hoc review as well as an inspection by the development team. The reviews will address the following issues and be guided by the checklist:
```

Comment:
Similar to the Design Verification Plan, I think this should be an internal endeavor rather than a stakeholder thing. Once again, stakeholders are generally only a knowledge source for requirements, they would be less familiar with verification and validation. Additionally, verification will also be done by classmates in the form of peer reviews so we can include that here too.

I think instead, you could bring up, in addition to the checklist, an internal walkthrough and comparison with the SRS, (link to SRS document), maybe HA (similarly link if included) and design documents since our tests are coming from those documents.

Reply:
peer reviews are walkthroughs

Reply:
I disagree with the second paragraph bc it's just a repeat of section 2 relevant documents